modules/sbo-rbac-requirements.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-rbac-requirements_{context}"]
+= RBAC requirements
```

Comment:
why do not we simply use the Section available in the upstream doc: https://redhat-developer.github.io/service-binding-operator/userguide/exposing-binding-data/rbac-requirements.html ?

Reply:
@pedjak We have used the section from the upstream doc. But in some places, we need to modify the content a bit to adhere to OpenShift/RH writing guidelines and templates that have sections for Introduction/concept parts and some such. HTH.